utils/mock-data-loader/portal/users.js

```diff
@@ -35,6 +35,16 @@ const USERS = [
     roles: ['maker'],
     bank: UKEF_TEST_BANK_1,
   },
+  {
+    username: 'BANK1_READ_ONLY',
+    password: 'AbC!2345',
+    firstname: 'B1 Read',
```

Comment:
Slight outside the scope of this PR, if time permits could you please align all the `firstname` and `surname` please with similar naming convention please.

Reply:
@abhi-markan do you mean rename all the mock users after their bank and role?

Reply:
Sure I've updated the name for this mock user. For the wider change, I'm worried about it causing tests to fail because of tests asserting on the names or looking up users by name, so I'll do that in a separate PR.